packages/website/_website/docs/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
I think the `_website` directory should be added to a gitignore

Reply:
those files are the 11ty output right?

Reply:
Yeap! They are 11ty output.